samples/python-fast-api/create_patient_trigger_webhook.py

```diff
@@ -58,7 +58,7 @@ class Address(pydantic.BaseModel):
     state: str = pydantic.Field(min_length=1)
     zip: str = pydantic.Field(min_length=5, max_length=5)
 
-    @pydantic.validator("zip", always=True)
+    @pydantic.validator("zip")
```

Comment:
@kevinwuhoo should this be a @pydantic.field_validator for pydantic v2?

Reply:
Yes sorry, you're right it should be:

```suggestion
    @pydantic.field_validator("zip")
```